services/resource-usage-tracker/tests/unit/with_dbs/test_api_licensed_items_checkouts.py

```diff
@@ -0,0 +1,139 @@
+# pylint:disable=unused-variable
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+# pylint:disable=too-many-arguments
+
+
+from datetime import UTC, datetime, timedelta
+from decimal import Decimal
+from typing import Generator
+
+import pytest
+import sqlalchemy as sa
+from models_library.api_schemas_resource_usage_tracker.licensed_items_checkouts import (
+    LicensedItemCheckoutGet,
+    LicensedItemsCheckoutsPage,
+)
+from models_library.resource_tracker_licensed_items_purchases import (
+    LicensedItemsPurchasesCreate,
+)
+from servicelib.rabbitmq import RabbitMQRPCClient
+from servicelib.rabbitmq.rpc_interfaces.resource_usage_tracker import (
+    licensed_items_checkouts,
+    licensed_items_purchases,
+)
+from simcore_postgres_database.models.resource_tracker_licensed_items_checkouts import (
+    resource_tracker_licensed_items_checkouts,
+)
+from simcore_postgres_database.models.resource_tracker_service_runs import (
+    resource_tracker_service_runs,
+)
+
+pytest_simcore_core_services_selection = [
+    "postgres",
+    "rabbit",
+]
+pytest_simcore_ops_services_selection = [
+    "adminer",
+]
+
+
+_USER_ID_1 = 1
+_WALLET_ID = 6
+
+
+@pytest.fixture()
+def resource_tracker_service_run_id(
+    postgres_db: sa.engine.Engine, random_resource_tracker_service_run
+) -> Generator[str, None, None]:
+    with postgres_db.connect() as con:
```

Comment:
Same here: check whether you can use something in `pytests_simcore.helpers.postgres_tools`